Project: Hello Microverse

gsmalik030 opened the pull request:
In this Project we:
Learn about basic workflow for GitHub.
we learned the uses and power of branches.
Added .gitignore file and learned how to ignore some files.
Added and configured Linters.

gsmalik030 committed:
Merge pull request #1 from gsmalik030/first-feature-branch

Create linters.yml

gsmalik030 committed:
Added html, css and test file

gsmalik030 committed:
Added Stylelint

gsmalik030 committed:
Fixed Stylelint

aimalamiri commented on .gitignore:
```diff
@@ -1,2 +1,3 @@
 git node_modules
```
- [ ] Good job, but this one has no impact, and it's highly recommended to remove the unnecessary lines of code from your files. It'll help you keep your repository cleaner. Kindly make sure to remove it.

gsmalik030 replied:
Thanks for highlighting.

aimalamiri commented on README.md:
```diff
@@ -8,7 +8,248 @@
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
- [ ] Looks great, but it's highly recommended to customize your README file and update the social media links so that people can contact you.

aimalamiri reviewed (changes requested):
Hi @gsmalik030, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![AlmostThereReachGIF](https://user-images.githubusercontent.com/34779928/191347504-9f70659c-a01f-48a1-ad46-b8722d46ffca.gif)

## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️

You did a great job, and your project is almost completed but we can call it done by fixing some small issues. Like It's required to write the summary of your task in the pull request, please make sure to update your pull request's summary and add enough information about the task you've done in the current pull request. Please read [the 5 basic rules to follow in Microverse projects](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) to learn more how to document your project professionally.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gsmalik030 commented:
In this Project we:
Learn about basic workflow for GitHub.
we learned the uses and power of branches.
Added .gitignore file and learned how to ignore some files.
Added and configured Linters.

gsmalik030 committed:
Changed social media links.

gsmalik030 committed:
Update README.md

gsmalik030 committed:
Updated .gitignore

gsmalik030 commented:
Good Day
Thank You, Sir
I changed it as you mentioned thanks for explaining and highlighting errors.

On Tue, Nov 29, 2022 at 7:32 PM Aimal Amiri ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @gsmalik030 <https://github.com/gsmalik030>,
>
> Good job so far!
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
> [image: AlmostThereReachGIF]
> <https://user-images.githubusercontent.com/34779928/191347504-9f70659c-a01f-48a1-ad46-b8722d46ffca.gif>
> Highlights
>
>    - Good job for using the right GitHub Flow 💯
>    - Awesome file organization 🗃️
>    - Commits are descriptive and meaningful 👍
>    - Linters are properly set up and passing 🟢 👍
>
> Required Changes ♻️
>
> You did a great job, and your project is almost completed but we can call
> it done by fixing some small issues. Like It's required to write the
> summary of your task in the pull request, please make sure to update your
> pull request's summary and add enough information about the task you've
> done in the current pull request. Please read the 5 basic rules to follow
> in Microverse projects
> <https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects>
> to learn more how to document your project professionally.
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In .gitignore
> <https://github.com/gsmalik030/Hello-Microverse/pull/2#discussion_r1035048223>
> :
>
> > @@ -1,2 +1,3 @@
>
>  git node_modules
>
>
>
>    - Good job, but this one has no impact, and it's highly recommended to
>    remove the unnecessary lines of code from your files. It'll help you keep
>    your repository cleaner. Kindly make sure to remove it.
>
> ------------------------------
>
> In README.md
> <https://github.com/gsmalik030/Hello-Microverse/pull/2#discussion_r1035049855>
> :
>
> > +- GitHub: ***@***.***(https://github.com/githubhandle)
>
> +- Twitter: ***@***.***(https://twitter.com/twitterhandle)
>
> +- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
>
>
>
>    - Looks great, but it's highly recommended to customize your README
>    file and update the social media links so that people can contact you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gsmalik030/Hello-Microverse/pull/2#pullrequestreview-1198009823>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3GIGD4D3ISXCTW6TZLAE4LWKY42BANCNFSM6AAAAAASOWJR7M>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Alfredbis29 reviewed (approved):
# STATUS: APPROVED  
 ## Hi @gsmalik030  ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:

![](https://camo.githubusercontent.com/54a5d803a138b5a1a1242a607e4c1192c852cab8c591c1ddcaa33d06b51560f6/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966)
 
 ## Optional suggestions
- [ ] N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
